<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckFrench</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_fileOpened">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>test_result</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_argument">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_argument">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>102</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Argument parser</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filepath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_process">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_updateResult">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>last_update</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_project">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="DataResultTableView" name="tableView_result">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>1081</width>
      <height>551</height>
     </rect>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Result"/>
    <addaction name="actionOpen_Recent_Result"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionKeybinding"/>
    <addaction name="actionTheme"/>
    <addaction name="actionCheck_for_Update"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionProjects"/>
    <addaction name="actionParsers"/>
    <addaction name="actionResults"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManage"/>
   <addaction name="menuView"/>
   <addaction name="menuPreferences"/>
  </widget>
  <action name="actionOpen_Recent_Result">
   <property name="text">
    <string>Open Recent Result</string>
   </property>
  </action>
  <action name="actionOpen_Result">
   <property name="text">
    <string>Open Result</string>
   </property>
  </action>
  <action name="actionProjects">
   <property name="text">
    <string>Projects</string>
   </property>
  </action>
  <action name="actionParsers">
   <property name="text">
    <string>Parsers</string>
   </property>
  </action>
  <action name="actionResults">
   <property name="text">
    <string>Results</string>
   </property>
  </action>
  <action name="actionKeybinding">
   <property name="text">
    <string>Keybinding</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="actionCheck_for_Update">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataResultTableView</class>
   <extends>QTableView</extends>
   <header>checkfrench/ui/widgets/dataresult_tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_project</tabstop>
  <tabstop>lineEdit_filepath</tabstop>
  <tabstop>lineEdit_argument</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
